### DISCUSSION TITLE
Payout not processing (created via API)

### DISCUSSION THREAD

**Discussion by Compoundxbt** [btcpayserver/btcpayserver#4714] [Technical Support]
Hi. I have 2 payouts in the "Awaiting Payment" status for the last day. They are not progressing. The server log outputs "2 found to process. Starting (and after will sleep for 00:05:00)". I had other payouts which processed automatically in the past, however, this issue is fairly new.

Pull Payment and Payout were created using v1 API.

Thanks in advance :)

**Comment by Kukks** [comment]
Anything in the logs?

**Reply by Compoundxbt** [reply]
![image](https://user-images.githubusercontent.com/126473923/221569473-9ba6cd58-ad71-46a3-930d-578a65b5508b.png)

Every 5 minutes in the logs: "2 found to process. Starting (and after will sleep for 00:05:00)"

**Reply by Kukks** [reply]
Do you have enough funds?

**Reply by Compoundxbt** [reply]
Yes I do. Funds > Payout amount.
